mscore/musescore.cpp

```diff
@@ -2967,6 +2967,12 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyRevisionButton->setIcon(*icons[int(Icons::copy_ICON)]);
+
+      copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
```

Comment:
hadrcoding the BR would mean to have several separate strings to translate rather than just one, and with context